waku/v2/protocol/waku_rln_relay/utils.nim

```diff
@@ -1026,9 +1025,11 @@ proc new*(T: type WakuRlnRelay,
   # relay protocol is the prerequisite of rln-relay
   if wakuRelay.isNil():
     return err("WakuRelay protocol is not mounted")
-  # check whether the pubsub topic is supported at the relay level
-  if conf.rlnRelayPubsubTopic notin wakuRelay.defaultPubsubTopics:
-    return err("The relay protocol does not support the configured pubsub topic")
+
+  # TODO: Review this. The Waku Relay instance is no longer keeping track of the default pubsub topics
+  # # check whether the pubsub topic is supported at the relay level
+  # if conf.rlnRelayPubsubTopic notin wakuRelay.defaultPubsubTopics:
+  #   return err("The relay protocol does not support the configured pubsub topic")
```

Comment:
@rymnc I need your input here. We no longer keep the "default pub-sub topics" as part of the `WakuRelay` instance. Would you like me to remove it?

Reply:
Yes, that works. As long as we can assert that the rln relay pubsub topic belongs to the configured pubsub topics, it should be ok